source/connect.txt

```diff
@@ -9,5 +9,174 @@ Connect to a Deployment
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+
+.. important::
+
+   Ensure that a MongoDB deployment is running before attempting to 
+   connect to it with ``mongosh``. The connection method you use 
+   depends on your deployment type.
+
+Local MongoDB Instance on Default Port
+--------------------------------------
+
+Run ``mongosh`` without any command-line options
+to connect to a MongoDB instance running on your **localhost** with 
+**default port** 27017:
+
+.. code-block:: sh
+
+   mongosh
```

Comment:
[optional] possibly add a second code-block after this one like, "This is equivalent to running the following command:

.. code-block:: sh

      mongosh "localhost:27017"